lnwallet/channel_test.go

```diff
@@ -3593,7 +3605,7 @@ func TestChanSyncFailure(t *testing.T) {
 	// Create a test channel which will be used for the duration of this
 	// unittest. The channel will be funded evenly with Alice having 5 BTC,
 	// and Bob having 5 BTC.
-	aliceChannel, bobChannel, cleanUp, err := CreateTestChannels()
+	aliceChannel, bobChannel, cleanUp, err := CreateTestChannels(false)
```

Comment:
What's the rationale behind doing some tweakless here, some not?

Reply:
I only do tweakless where it would materially affect the test. Alternatively, everything can just be wrapped into one giant for loop and always test both behavior. I'm not sure if this is worth it though since we test the basic channel open an update functionality for both, while many tests just repeat that logic then asserts some db transitions that are unrelated to the commitment format.

Reply:
Interesting. It would be useful to do it at least once do, to make sure it all passes.

I'm not opposed to wrapping them all in a for loop, it seems like free test coverage of the tweakless channels without much effort.

Reply:
Would also let use smoothly transition to remove the tweaked format later, as all tests would be running for the new format.